Sources/Component/Action/ActionView.swift

```diff
@@ -57,9 +64,37 @@ public final class ActionView<ContentView: StatefulViewProtocol>: StatefulView<A
     @objc
     private func updateBackgroundColor() {
         if [.highlighted, .selected].contains(button.state) {
-            button.backgroundColor = UIColor.black.withAlphaComponent(0.15)
+            switch model.highlightAnimation {
+            case .normal:
+                button.backgroundColor = UIColor.black.withAlphaComponent(0.15)
+
+            case .curveEaseInOut:
+                UIView.animate(
+                    withDuration: model.animationDuration,
+                    delay: 0,
+                    options: [.curveEaseInOut],
+                    animations: {
+                        self.view.transform = CGAffineTransform(scaleX: 0.95, y: 0.95)
+                    },
+                    completion: nil
+                )
+            }
         } else {
-            button.backgroundColor = UIColor.black.withAlphaComponent(0)
+            switch model.highlightAnimation {
+            case .normal:
+                button.backgroundColor = UIColor.black.withAlphaComponent(0)
+
+            case .curveEaseInOut:
+                UIView.animate(
```

Comment:
Same as above, think about a more generic way to introduce specific highlighting states.

Reply:
See comment above.

Reply:
Done.